### PR TITLE
[interfaces/_pybullet.py] add timeout to wait_interpolaiton()

### PR DESCRIPTION
**Pull request by ktro4989**
Update wait_interpolation in skrobot/interfaces/_pybullet.py with timeout
please review it!

### CODE REVIEW

**Comment by iory** [comment]
Thanks for your PR!
Could you rebase your commit?
```git pull --rebase origin master```

File: skrobot/interfaces/_pybullet.py [hunk 1]
```diff
@@ -289,13 +273,15 @@ def wait_interpolation(self, thresh=0.05):
                                                     idx)
                 if abs(velocity) > thresh:
                     wait = True
+                if time.time() - start > timeout:
+                    wait = False
```
Comment (iory) [inline skrobot/interfaces/_pybullet.py:277]: If this function is timed out, should return `False`.

Comment (ktro4989) [reply skrobot/interfaces/_pybullet.py:277]: ok, i got it

File: skrobot/interfaces/_pybullet.py [hunk 2]
```diff
@@ -8,6 +8,8 @@
 from skrobot.coordinates import matrix2quaternion
 from skrobot.coordinates import quaternion2rpy
 
+import time
```
Comment (iory) [inline skrobot/interfaces/_pybullet.py:11]: > Imports should be grouped in the following order:
> standard library imports
> related third party imports
> local application/library specific imports

https://www.python.org/dev/peps/pep-0008/#imports
https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement

File: skrobot/interfaces/_pybullet.py [hunk 3]
```diff
@@ -291,6 +295,9 @@ def wait_interpolation(self, thresh=0.05):
                     wait = True
             if wait is False:
                 break
+            if time.time() - start > timeout:
+                return False
+            time.sleep(1)
```
Comment (iory) [inline skrobot/interfaces/_pybullet.py:300]: Why did you add `time.sleep(1)`?

**Comment by iory** [comment]
This repository uses Travis to check codes. So if those Travis tests don't pass, I could not merge this PR.
Could you check the Travis?

image

File: skrobot/interfaces/_pybullet.py (hunk 3 above)
Comment (ktro4989) [reply skrobot/interfaces/_pybullet.py:300]: sorry, I dont't need it 
I thought I need to add `time.sleep(1)` to make 1-loop  loop in 1sec

Comment (iory) [reply skrobot/interfaces/_pybullet.py:300]: OK. please delete it.

**Comment by ktro4989** [comment]
i got the other error i didin't touch
[https://travis-ci.com/github/iory/scikit-robot/jobs/351967829](https://travis-ci.com/github/iory/scikit-robot/jobs/351967829)

**Comment by iory** [comment]
OK. I checked the error #138 .
By the way, please change this PR's title so that you can understand the purpose of this PR.

**Comment by iory** [comment]
Please rebase and check the Travis test.

**Comment by ktro4989** [comment]
finnally i passes test, sorry I imported liblaries wrong order

**Comment by iory** [comment]
LGTM!
